packages/react-native/android/src/main/java/com/dashx/sdk/DashXModule.kt

```diff
@@ -1,9 +1,12 @@
 package com.dashx.sdk
 
-import com.facebook.react.bridge.*
+import com.facebook.react.bridge.ReactApplicationContext
```

Comment:
Removing as per https://jiga.blog/2020/07/12/avoiding-wildcard-imports-in-java-kotlin-with-intellij/